chia/cmds/farm_funcs.py

```diff
@@ -232,6 +233,10 @@ async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int,
     minutes = -1
     if blockchain_state is not None and plots is not None:
         proportion = total_plot_size / blockchain_state["space"] if blockchain_state["space"] else -1
-        minutes = int((await get_average_block_time(rpc_port) / 60) / proportion) if proportion else -1
```

Comment:
There seems to be a but in this old code, because of the `-1` defaults. If `blockchain_state["space"]` counts as void, then `proportion` is `-1`, but since `-1` does not as void, one will get a division by `-1` here.

Reply:
this seems like it should be a separate pr

Reply:
Yes, true, just pointing out the bug in existing code